GliaWidgets/Sources/Component/Button/Header/HeaderButton.Props.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+extension HeaderButton {
+    struct Props {
+        var tap: Cmd
```

Comment:
Agree, however, it is not possible to do until [EngagementViewController](https://github.com/salemove/ios-sdk-widgets/pull/485/files#diff-ae805a97a762b66a6e90c10388387a488324c2f29180305c3bd12b5e8f765fa0) refactored, because it still doesn't use data driven approach, so to don't add that changes to this PR we still need to have vars here.

Reply:
I think for [EngagementViewController](https://github.com/salemove/ios-sdk-widgets/pull/485/files#diff-ae805a97a762b66a6e90c10388387a488324c2f29180305c3bd12b5e8f765fa0) we will just form props inside it once without refactoring it drastically for data-driven. Let's discuss it for clarity.